import mysql.connector
from mysql.connector import errorcode

config = {
    "user": "pysports user",
    "password": "fender09",
    "host": "127.0.0.1",
    "database": "pysports",
    "raise_on_warnings": True
}